pvl/decoder.py

```diff
@@ -109,6 +137,14 @@ def raise_unexpected(self, stream, token=None):
     def raise_unexpected_eof(self, stream):
         self.raise_error('Unexpected EOF', stream)
 
+    def broken_parameter_value(self, lineno):
```

Comment:
A better name would be ``broken_assignment`` but Parameter (more specifically Parameter Name) and Value to come from  the PVL spec.